lib/clients/rtm/client.js

```diff
@@ -617,6 +628,35 @@ RTMClient.prototype.sendMessage = function sendMessage(text, channelId, optCb) {
   }, optCb);
 };
 
+/**
+ * Helper for updating a sent message via the 'chat.update' API call
+ * @param message {object} message The message object to be updated,
+ * see /lib/clients/web/facets/chat.js for more details
+ * @param {Function=} optCb Optional callback
+ */
+RTMClient.prototype.updateMessage = function updateMessage(message, optCb) {
+  var err;
+  var ret;
+
+  if (this.connected) {
+    ret = this._chat.update(message.ts, message.channel, message.text, message.opts, optCb);
```

Comment:
Could you say a little more about this design decision? Connection to the MS isn't necessary for a call to chat.update to succeed, being a web API call.